nixos.nix

```
@@ -87,11 +87,11 @@ in
 
     system.activationScripts =
       let
-        # Create a directory in persistent storage, so we can bind
+        # Script to create a directory in persistent storage, so we can bind
         # mount it. The directory structure's mode and ownership mirror those of
         # persistentStoragePath/dir;
         # TODO: Move this script to it's own file, add CI with shfmt/shellcheck.
-        mkDirWithPerms = persistentStoragePath: dir:
+        create-directories = pkgs.writeShellScript "impermanence-create-directories"
```

```suggestion
        createDirectories = pkgs.writeShellScript "impermanence-create-directories"
```
To keep it in line with the `nixpkgs` variable naming style.